cmip6_downscaling/workflows/bcsd_flow.py

```diff
@@ -55,32 +56,42 @@
 get_coarse_obs_task = task(
     get_coarse_obs,
     tags=['dask-resource:TASKSLOTS=1'],
+    max_retries=5,
```

Comment:
what do the `max_retries` and `retry_delay` arguments do? is it to give it time to get created before trying to delete it?